onnxruntime/core/providers/nnapi/nnapi_builtin/builders/helper.cc

```diff
@@ -476,9 +476,11 @@ bool IsInputSupported(const NodeArg& input, const std::string& parent_name) {
 
 std::vector<std::vector<size_t>> GetSupportedNodes(const GraphViewer& graph_viewer, const OpSupportCheckParams& params) {
   std::vector<std::vector<size_t>> supported_node_groups;
-  if (params.android_sdk_ver < ORT_NNAPI_MIN_API_LEVEL) {
-    LOGS_DEFAULT(WARNING) << "All ops will fallback to CPU EP, because Android API level [" << params.android_sdk_ver
-                          << "] is lower than minimal supported API level [" << ORT_NNAPI_MIN_API_LEVEL
+  if (params.android_feature_level < ORT_NNAPI_MIN_API_LEVEL) {
```

Comment:
>ORT_NNAPI_MIN_API_LEVEL [](http://example.com/codeflow?start=37&length=23)

nit: should ORT_NNAPI_MIN_API_LEVEL be ORT_NNAPI_MIN_FEATURE_LEVEL?

Reply:
This is a build.py param, also for example ANEURALNETWORKS_FEATURE_LEVEL_1 = 27, for this should we use 1 or 27? It will be a bit confusing